### DISCUSSION TITLE
DNS redirect changes name to ip.

### DISCUSSION THREAD

**Discussion by mmorales99** [fastapi/fastapi#6229] [Questions] [answered]
### First Check

- [x] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi.responses import Response, RedirectResponse

app = FastAPI()

## Start de la web
@app.get('/')
async def root():
  return Response(content="<html><head></head><body><h1>HELLO WORLD!!</h1></body></html>",status_code=HTTPStatus.OK)

if __name__ == "__main__":
    uvicorn.run("main:app",
                host="0.0.0.0",
                port=12345,
                workers=1,
                app_dir=os.path.join(".","SRC"),
                server_header=False,
                # ssl-keyfile=os.path.join('.','cert','key.pem')
                # ssl-certfile=os.path.join('.','cert','cert.pem')
                )
```

### Description

I know the question is nooblike, but i'm getting verry frustrated with this.

I have my vps(OVH cheapest one) where the above code is running.
I have my domain name already buyed.
I have my dns(also OVH cheapest one) and configured to point the domain name to the IP of the VPS.
When i access my 'web page' through my domain it lands correctly to the page, but in the to bar of google chrome shows the ip instead the domain name.

I have configured Caddy and Uvicorn with the Let's encrypt certificate.

Am I missing something??

Versions:
OS: ubuntu 22.04 x64
Python: 3.10.4
Caddy : Latest
Uvicorn: 0.17.6
FastApi: 0.77.1

### Operating System

Linux

### Operating System Details

ubuntu 22.04 x64

### FastAPI Version

0.77.1

### Python Version

3.10.4

**Comment by JarroVGIT** [comment]
How did you configure your DNS? Did you set an "A record" to your public IP address?

This doesn't seem an FastAPI issue by the way, I think it is more appropriate to ask this on SO.

**Comment by mmorales99** [comment, marked as answer]
I feel very dumb... some of my colleges drive me to confusion since they told me it had an option that i had to change... they were talking about wordpress
Thank you so much!! And sorry for the dumb question...
Gonna lay this video here for the posterity, its tiangolo explaining the whole process, idial thing to be in the uvicorn/fastapi documentation web: https://www.youtube.com/watch?v=7N5O62FjGDc

**Comment by JarroVGIT** [comment]
Never feel dumb, be happy you have learned something tangible today! :)
